Source/DlgSystem/DlgConditionCustom.h

```diff
@@ -29,7 +29,7 @@ class DLGSYSTEM_API UDlgConditionCustom : public UDlgObject
 	FString GetEditorDisplayString(UDlgDialogue* OwnerDialogue, FName ParticipantName);
 	virtual FString GetEditorDisplayString_Implementation(UDlgDialogue* OwnerDialogue, FName ParticipantName)
 	{
-		return FString(TEXT("[")) + ParticipantName.ToString() + FString(TEXT("] ")) + GetName();
+		return FString(TEXT("[")) + ParticipantName.ToString() + FString(TEXT("] ")) + GetClass()->GetDisplayNameText().ToString();
```

Comment:
You sure this compiles in non editor builds? 
![image](https://github.com/NotYetGames/DlgSystem/assets/1269608/d5861a61-f5d0-4135-96e9-5309d72ea236)

Reply:
My bad, that got past me. I could probably add an #if WITH_EDITOR inside the GetEditorDisplayString_Implementation itself to return GetDisplayNameText() or the GetName(), but I'll just leave it to you guys if you want to add something like that.